src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -1296,34 +1296,23 @@ public LazyConditionalChoiceExpression(Lazy<IOperation> condition, Lazy<IOperati
     /// </summary>
     internal abstract partial class BaseConversionExpression : Operation, IHasOperatorMethodExpression, IConversionExpression
     {
-        protected BaseConversionExpression(ConversionKind conversionKind, bool isExplicit, bool usesOperatorMethod, IMethodSymbol operatorMethod, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue) :
+        protected BaseConversionExpression(CommonConversion conversion, bool isExplicitInCode, bool throwsExceptionOnFailure, bool isChecked, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue) :
                     base(OperationKind.ConversionExpression, syntax, type, constantValue)
         {
-            ConversionKind = conversionKind;
-            IsExplicit = isExplicit;
-            UsesOperatorMethod = usesOperatorMethod;
-            OperatorMethod = operatorMethod;
+            Conversion = conversion;
+            IsExplicitInCode = isExplicitInCode;
+            ThrowsExceptionOnFailure = throwsExceptionOnFailure;
+            IsChecked = isChecked;
         }
-        /// <summary>
-        /// Value to be converted.
-        /// </summary>
+
         public abstract IOperation Operand { get; }
-        /// <summary>
-        /// Kind of conversion.
-        /// </summary>
-        public ConversionKind ConversionKind { get; }
-        /// <summary>
-        /// True if and only if the conversion is indicated explicity by a cast operation in the source code.
-        /// </summary>
-        public bool IsExplicit { get; }
-        /// <summary>
-        /// True if and only if the operation is performed by an operator method.
-        /// </summary>
-        public bool UsesOperatorMethod { get; }
-        /// <summary>
-        /// Operation method used by the operation, null if the operation does not use an operator method.
-        /// </summary>
-        public IMethodSymbol OperatorMethod { get; }
+        public CommonConversion Conversion { get; }
+        public bool IsExplicitInCode { get; }
+        public bool ThrowsExceptionOnFailure { get; }
+        public bool IsChecked { get; }
+        public abstract string LanguageName { get; }
+        public bool UsesOperatorMethod => Conversion.IsUserDefined;
+        public virtual IMethodSymbol OperatorMethod => Conversion.MethodSymbol;
```

Comment:
It was virtual because it needed to be redefined in the C# version, but that's not necessary any more. I'll remove it.